src/js/common/messageHelper.js

```diff
@@ -12,19 +12,25 @@ var gpii   = fluid.registerNamespace("gpii");
 
 fluid.registerNamespace("gpii.handlebars.helper.messageHelper");
 
-gpii.handlebars.helper.messageHelper.getHelper = function () {
+gpii.handlebars.helper.messageHelper.getHelper = function (that, serverAware) {
     return function (messageKey, dataOrRootContext, rootContext) {
         if (arguments.length < 2) {
             fluid.fail("You must call the 'messageHelper' helper with at least a message key.");
         }
         else {
-            // Pick up the message bundles from the root context, which is always the last argument.
-            var messages = fluid.get(rootContext || dataOrRootContext, "data.root.messages");
-
+            var messages;
+            if (serverAware) {
+                messages = fluid.get(serverAware, "model.messages");
+            }
+            else {
+                // Currently, the server verion of the renderer is still storing the messages in it's
+                // options area, so if no serverAware is wired in, we'll assume this is running in node.js
+                messages = fluid.get(rootContext || dataOrRootContext, "data.root.messages");
+            }
             // If we have a third argument, then the second argument is our "data".  Otherwise, we use the root context (equivalent to passing "." as the variable).
-            var data = rootContext ? dataOrRootContext : fluid.get(dataOrRootContext, "data.root");
+            // TODO var data = rootContext ? dataOrRootContext : fluid.get(dataOrRootContext, "data.root");
```

Comment:
OK, this is another case where you've neutered a feature without any comment on why, and left the old functionality commented out.  What's your thinking here?